gce_workload_cert_refresh/main.go

```diff
@@ -29,7 +29,7 @@ import (
 )
 
 const (
-	contentDirPrefix  = "/run/secrets/workload-spiffe-contents"
+	contentDirPrefix  = "/run/secrets/.workload-spiffe-contents"
```

Comment:
no need to do this, this is a convenience for directories and files in the user's home directory. it would be confusing to use that convention here.

Reply:
yes, please revert